docs/addons/install-addons.md

```diff
@@ -35,40 +59,51 @@ Next, update [`.storybook/main.js|ts`](../configure/index.md#configure-story-ren
 
 <!-- prettier-ignore-end -->
 
-<Callout variant="info">
+When you run Storybook, the accessibility testing addon will be enabled.
 
-Addons may also require addon-specific configuration. Read their respective READMEs.
+![Storybook addon installed and registered](./storybook-addon-installed-registered.png)
 
-</Callout>
+### Removing addons
 
-Now when you run Storybook the accessibility testing addon will be enabled.
+To remove an addon from Storybook, you can choose to manually uninstall it and remove it from the configuration file (i.e., [`.storybook/main.js|ts`](../configure/index.md)) or opt-in to do it automatically via the CLI with the [`remove`](../api/cli-options.md#remove) command. For example, to remove the [Accessibility addon](https://storybook.js.org/addons/@storybook/addon-a11y) from Storybook with the CLI, run the following command:
 
-![Storybook addon installed and registered](./storybook-addon-installed-registered.png)
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-remove-command.yarn.js.mdx',
+    'common/storybook-remove-command.npm.js.mdx',
+    'common/storybook-remove-command.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
 
-### Using preset addons
+## Using preset addons
```

Comment:
We shouldn't document `preset-scss`. It's unmaintained and was last published in 2020

Reply:
I'm also not sure whether it makes sense to call out preset addons specifically as a concept anymore. An addon is just an addon?

Reply:
@shilman by looking at the presets we currently have, the only one we provide is the [preset-create-react-app](https://github.com/storybookjs/presets/tree/master/packages/preset-create-react-app); if a user installs Storybook based on Create React App tooling. More than ok to switch it over to that one if you're ok with it. And with that, do you know if the documented options are still valid? 

About the naming convention for presets, we can just follow up on this later since we'll need to adjust this part and the rest of the documentation. Do you think that will work for you?

Reply:
@shilman, as we discussed, I've removed the presets addons section to avoid documenting a deprecated preset so we can start aligning towards the naming convention based on my comment above.